integration/test/basic/basic_test.go

```diff
@@ -57,3 +58,46 @@ func TestChartOperatorBootstrap(t *testing.T) {
 		t.Fatalf("bad release status, want %q, got %q", expectedStatus, actualStatus)
 	}
 }
+
+// TestChartConfigChartsInstalled checks that the charts for any chartconfig
+// CRs installed in the cluster have been deployed.
```

Comment:
New e2e test that checks that the charts for any related chartconfigs have been deployed.